bot/src/main/java/edu/java/bot/commands/Command.java

```diff
@@ -1,7 +1,20 @@
 package edu.java.bot.commands;
 
 import com.pengrad.telegrambot.model.Update;
+import edu.java.bot.model.Bot;
+import lombok.Getter;
 
-public interface Command {
-    void execute(Update update);
+@Getter
+public abstract class Command {
+    private final String name;
+    private final String description;
+    private final Bot bot;
+
+    protected Command(String name, String description, Bot bot) {
+        this.name = name;
+        this.description = description;
+        this.bot = bot;
+    }
+
+    abstract public void execute(Update update);
```

Comment:
Проблема: порядок модификаторов не соответствует Java-спецификации.

Предложение:

```suggestion
    public abstract void execute(Update update);
```